packages/headless/src/controllers/smart-snippet-questions-list/headless-smart-snippet-questions-list.ts

```diff
@@ -33,12 +28,28 @@ export interface SmartSnippetQuestionsList extends Controller {
   /**
    * Expand the specified snippet suggestion.
    *
+   * @param identifier - The `questionAnswerId` of the smart snippet to expand.
+   */
+  expand(identifier: string): void;
+  /**
+   * Expand the specified snippet suggestion.
+   *
+   * @deprecated - Use expand(identifier: string) instead.
```

Comment:
I tested, and when we try to call `expand` with an object, vscode now draws a strikethrough on the function. However, when we put a string, it doesn't draw a strikethrough. Pretty nice behaviour.